install/ui/src/freeipa/policy.js

```diff
@@ -72,6 +72,10 @@ return {
                         {
                             name: 'cospriority',
                             required: true
+                        },
+                        {
+                            name: 'passwordgracelimit',
+                            measurement_unit: 'number_of_passwords'
```

Comment:
The unit isn't quite right. It defines the number of LDAP BIND that are allowed post-expiration so that the password can be reset by the user. I'm struggling with a pity wording for it though :-) maybe "grace logins" is sufficient.

Reply:
After deciding on the wording that must be added to the i18n messages dict:

https://github.com/freeipa/freeipa/blob/c2e79fa0b64be5d51d3ef0e70bb8671cbe803657/ipaserver/plugins/internal.py#L417-L420

And i18n object used for testing:

https://github.com/freeipa/freeipa/blob/c2e79fa0b64be5d51d3ef0e70bb8671cbe803657/install/ui/test/data/i18n_messages.json#L214-L217

Reply:
"Grace logins" sounds good to me, I will do the changes to the files mentioned above. If we come up with a better name, I can change it later :)